ferrocene/doc/internal-procedures/src/working-with-the-ci.rst

```diff
@@ -188,6 +183,39 @@ of the job:
    ./x clean
    ./x --stage 2 dist rust-std
 
+Preserving build artifacts
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you wish to preserve your ``build/`` artifacts, it may make sense to
+re-clone the Ferrocene repository inside the container or mount a dedicated
+folder for ``/ferrocene/build``:
+
+.. code-block:: bash
+
+   docker run --rm --tty --interactive --workdir /ferrocene \
+      --mount "type=bind,src=$(pwd),dst=/ferrocene" \
+      --mount "type=bind,src=$(pwd)/container-build,dst=/ferrocene/build" \
+      ubuntu-20 bash
+
+Note that you will need to create the `container-build` folder beforehand.
+
+Sharing miscelaneous folders with the container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need to share any extra directories from the host with the container you
+can use a similar strategy as the one used to mount the ferrocene repository.
+
+For example, if you want to share the host `~/.aws` folder configuration so you
+can authenticate in the container run:
+
+.. code-block:: bash
+
+   docker run --rm --tty --interactive --workdir /ferrocene \
+      --mount "type=bind,src=$(pwd),dst=/ferrocene" \
+      --mount "type=bind,src=$HOME/.aws,dst=/home/ci/aws" \
+      ubuntu-20 bash
+
+Remember that any folders mounted with the `bind` type must exist beforehand.
```

Comment:
should clarify if the folders should exist on host or on guest

Reply:
done